src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -55,6 +55,8 @@ pub struct TargetInfo {
     pub rustflags: Vec<String>,
     /// Extra flags to pass to `rustdoc`, see [`extra_args`].
     pub rustdocflags: Vec<String>,
+    /// Whether or not rustc (stably) supports the `--check-cfg` flag.
```

Comment:
I believe this could be removed once `--check-cfg` is stabilized. Could we add a comment tracking this?

```suggestion
    /// Whether or not rustc (stably) supports the `--check-cfg` flag.
    ///
    /// Can be removed once it is stabilized (presumably 1.80.0)
```

Alternatively, we can create an issue tracking what to update after we hit 1.80.0.

Reply:
`--check-cfg` is already stabilized. This option exists so that Cargo can support older version of `rustc` where it isn't yet stable.

I added a comment reflecting that.